AdaptiveUICodelab/app/build.gradle

```diff
@@ -81,8 +81,9 @@ dependencies {
     implementation 'androidx.core:core-ktx:1.9.0'
     implementation "androidx.window:window:1.0.0"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4"
+    implementation "androidx.compose.material3:material3-window-size-class:1.0.1"
```

Comment:
This Gradle is using BOM for Compose, you can omit the version and let the BOM do its job. Also maybe consider moving this line next to material3 for better grouping and readability IMO

Reply:
Thanks for your help, I didn't know that information. The project had classes that were not compiling and looking through the package import I realized that this library was missing. One more learning. Thanks

Reply:
Hi it was intentionally removed and to be added as a part of the codelab learning. It should not be giving compiling issues I will look into it. For now I will close this PR as it's not needed.